### PR TITLE
Mobile Nav Bar

### PR DESCRIPTION
**Pull request by lisarawlings**

### CODE REVIEW

**Review by jadavey91 — APPROVED** [review]

**Review by Riley142 — APPROVED** [review]

File: app/src/App.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import React from "react";
 import "./App.css";
+import ExampleComponent from "./Components/ExampleComponent";
+import Header from "./Components/Header";
+import InteractionBar from "./Components/InteractionBar";
+import Messages from "./Components/Messages";
+import MobileNavBar from "./Components/MobileNavBar";
 import TweetCompose from "./Components/TweetCompose"
 import Header from "./Components/Header";
```
Comment (boothscript) [inline app/src/App.js:7]: Header has been imported twice so will fail ESLint check

File: app/src/App.js [hunk 2]
```diff
@@ -9,6 +14,13 @@ import Feed from "./Components/Feed";
 function App() {
   return (
     <div className="App">
+
+      <ExampleComponent />
+      <Header />
+      <i className="ri-notification-3-line" />
+      <InteractionBar />
+      <Messages />
+      <MobileNavBar />
```
Comment (boothscript) [inline app/src/App.js:17]: You'll need to remove `<Messages />`, `<i className="ri-notification-3-line" /> `and ` <ExampleComponent />` for now.

File: app/src/Components/Messages.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Collapsible from "react-collapsible";
+import "./Messages.css";
+
+const Messages = () => {
+  return (
+    <div className="flexContainer">
+      <Collapsible id="Messages" trigger="Messages" i className="ri-mail-line">
```
Comment (boothscript) [inline app/src/Components/Messages.js]: it looks like you deleted an icon but left some of the code in the Collapsible tag

**Review by boothscript — CHANGES_REQUESTED** [review]
There are a couple of things that need to be changed on this before it can be merged.

**Review by boothscript — APPROVED** [review]

**Pull request merged**